jsonschema-core.xml

```diff
@@ -219,6 +223,12 @@
                             <t hangText="string:">A string of Unicode code points, from the JSON "string" value</t>
                         </list>
                     </t>
+                    <t>
+                        Binary data MAY be treated as an instance, however no data type in the data model
+                        is suitable.  Therefore, only schemas such as the empty schema that do not
+                        constrain the type can be considered to pass.  Rationales for and behavior of
```

Comment:
I _think_ commas may be required: "Rationales for, and behavior of,"

Reply:
Not in this instance.